package.json

```diff
@@ -13,14 +13,14 @@
   },
   "dependencies": {
     "@financial-times/n-logger": "^10.3.0",
-    "isomorphic-fetch": "^2.0.0",
-    "n-eager-fetch": "^5.1.0"
+    "isomorphic-fetch": "^3.0.0",
+    "n-eager-fetch": "^7.0.0"
```

Comment:
I feel medium confident in this change. In order to adopt n-eager-fetch v7 we need to bump isomorphic-fetch to v3 to match it. From what I can tell there aren't significant code changes between the two apart from bumping to [node-fetch v2](https://github.com/node-fetch/node-fetch/blob/main/docs/v2-UPGRADE-GUIDE.md).

We're releasing a major version anyway so I can just document this change in the release notes.